flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -382,7 +388,8 @@ private static void checkAndCreateDirectory(File directory) throws IOException {
 				nativeMetricOptions,
 				metricGroup,
 				restoreStateHandles,
-				ttlCompactFiltersManager);
+				ttlCompactFiltersManager,
+				writeBufferManagerCapacity);
```

Comment:
```suggestion
				optionsContainer.getWriteBufferManagerCapacity();
```
We can directly use `optionsContainer.getWriteBufferManagerCapacity()` here and don't need to pass `writeBufferManagerCapacity` through constructor parameters.

Reply:
True, better that wau.